### DISCUSSION TITLE
[Simulateur salarié] Contrat unique d'insertion

### DISCUSSION THREAD

**Discussion by johangirod** [betagouv/mon-entreprise#2628] [Idées]
Il faudrait ajouter la prise en compte des CUI dans le simulateur salarié. Il en existe deux type différents : les CUI-CAE pour les associations et les CUI-CIE pour les entreprises.

Concrètement, cela prends la forme d'une aide financière directe à l'entreprise, exprimée en pourcentage du SMIC

### ~Exonération de cotisations patronales~

EDIT : Il semblerait que l'exonération soit supprimée depuis le 1er janvier 2019
> https://www.urssaf.fr/portail/home/employeur/beneficier-dune-exoneration/exonerations-ou-aides-liees-a-la/le-contrat-unique-dinsertion-cui.html

### Aide à l'embauche

Le problème est que l'aide est régionale et est fixée par arrêté préfectoral. Et je n'arrive pas à mettre la main sur la liste des taux par région. Je pense même que c'est impossible, et que seul les conseillers pole emploi peuvent répondre à cette question. On pourrait imaginer ajouter une question demandant le % du SMIC, mais on perd un peu l'intérêt...

> Le montant de l’aide de l’État, exprimé en taux de prise en charge par rapport au SMIC, est donc différent selon les régions, en fonction des besoins spécifiques du bassin d’emploi. Le conseiller Pôle emploi ou en mission locale est en mesure d’apporter des précisions sur le montant et la durée de l’aide qui résultent des arrêtés régionaux.

**Pour les CAE**
> L’aide mensuelle versée est fixée au niveau régional par des arrêtés des préfets de région, et s’applique dans la limite de 95% du taux horaire brut du Smic. Cette aide est modulable en fonction de la situation du bénéficiaire, de l’employeur et des spécificités du marché de l’emploi.

**Pour les CIE**
> L’aide ne peut excéder 47% du montant brut du SMIC par heure travaillée, dans la limite de la durée légale hebdomadaire du travail : les taux de prise en charge déterminant son montant sont fixés par un arrêté du préfet de région, en fonction des critères énumérés ci-dessus et compte tenu, le cas échéant, des statistiques publiques de l’emploi dans la région.

Sources :
- https://travail-emploi.gouv.fr/emploi/parcours-emploi-competences/cui-cae (association)
- https://travail-emploi.gouv.fr/emploi/parcours-emploi-competences/cui-cie (entreprises)
- https://www.urssaf.fr/portail/home/employeur/beneficier-dune-exoneration/exonerations-ou-aides-liees-a-la/le-contrat-unique-dinsertion-cui.html

---
A vérifier si ça un rapport mais je suis tombé par hasard sur cette page qui précise les exonérations pour les SIAE 
https://www.urssaf.fr/portail/home/employeur/beneficier-dune-exoneration/exonerations-ou-aides-liees-au-s/les-structures-dinsertion-par-la/les-associations-intermediaires/exonerations.html

**Comment by mquandalle** [comment]
CUI-CAE ça couvre l'ensemble du sujet de #162 ?

**Comment by johangirod** [comment]
Bien vu ! Oui, à priori ça couvre l'ensemble du sujet. Sauf que la seule limite qui reste aujourd'hui est celle-ci : 
> trouver une bonne source de données sur les pourcentages d'aide au salaire qui dépendent d'arrêtés régionaux
Exemple : https://blogs.alternatives-economiques.fr/abherve/2018/03/03/en-normandie-un-arrete-prefectoral-sur-les-pec-assez-restrictif

Les autres sont à priori réglée

**Comment by mquandalle** [comment]
On trouve les arrêtés sur le site de la DIRRECTE de chaque région avec une [recherche Google](https://www.google.com/search?hl=fr&biw=1920&bih=976&sxsrf=ALeKk02z1g5zx9dXIwldGT6I3UHfTyW1cQ%3A1604672821690&ei=NV2lX-ycKeLkgwffmou4Cg&q=arrete+emploi+comp%C3%A9tence+site%3Adireccte.gouv.fr&oq=arrete+emploi+comp%C3%A9tence+site%3Adireccte.gouv.fr&gs_lcp=CgZwc3ktYWIQAzoECAAQR1CUOliRbGDSbWgDcAN4AIABsAGIAakLkgEEMTcuNJgBAKABAaoBB2d3cy13aXrIAQjAAQE&sclient=psy-ab&ved=0ahUKEwisi5GMkO7sAhVi8uAKHV_NAqcQ4dUDCAw&uact=5) mais le format n'est pas standardisé. Parfois les taux sont repris sur le site, mais le plus souvent ça ne redirige que vers un scan pdf de l'arrêté: ex [bretagne](http://bretagne.direccte.gouv.fr/sites/bretagne.direccte.gouv.fr/IMG/pdf/2020_10_09_arr_aides_cui.pdf), [grand est](http://grand-est.direccte.gouv.fr/sites/grand-est.direccte.gouv.fr/IMG/pdf/arrete_prefectoral_pec_sign.pdf).

Ceci étant dit j'ai l'impression que dans le cadre du plan de relance (hashtag 1 jeune 1 solution), il y a un taux unique de 65% au niveau national http://pays-de-la-loire.direccte.gouv.fr/Contrats-aides-arrete-relatif-aux-parcours-emploi-competences, que l'on retrouve dans les arrêtés PDF ci-dessus. Ça on doit pouvoir l'intégrer !

**Comment by johangirod** [comment]
> Ceci étant dit j'ai l'impression que dans le cadre du plan de relance (hashtag 1 jeune 1 solution), il y a un taux unique de 65% au niveau national http://pays-de-la-loire.direccte.gouv.fr/Contrats-aides-arrete-relatif-aux-parcours-emploi-competences, que l'on retrouve dans les arrêtés PDF ci-dessus. Ça on doit pouvoir l'intégrer !

Ah oui, bien vu, ça serait top ! On a même la circulaire DGEFP qui rends tout ça officiel : 
> https://circulaires.legifrance.gouv.fr/circulaire/id/45068

J'enlève le en attente / réflexion

Ceci étant dit, ça serait pas mal de prendre contact avec l'organisme qui s'occupe de ce dispositif (visiblement c'est la DGEFP)

**Comment by mquandalle** [comment]
Intéressant à prioriser début 2021, on a la possibilité d'être relayé sur https://www.1jeune1solution.gouv.fr/je-recrute/articles

**Comment by mquandalle** [comment]
TODO simulateur PEC-jeune :

- [ ] Intégrer l'aide = 65% × brut, avec les conditions d'applicabilité correspondantes (temps du travail, salaire)
- [ ] Message d'avertissement : Le PEC est réservé au secteur non marchand (associations, secteur public)
- [ ] La durée de l'aide dépend de chaque région (les fameux decrets en pdf cités plus haut). Pole Emploi à une liste exhaustive qui normalement est à jour, mais il y a souvent des changements et ça reste des pdf. Donc l'idée est d'afficher un avertissement « La durée de l'aide dépend de votre région, rapprochez-vous d'un conseiller »
- [ ] Exonération de certaines cotisations patronales https://www.legisocial.fr/contrat-de-travail/embauche/aides-regime-social-pec-2019.html#toc-cotisations-patronales (à vérifier)

**Comment by Juliedelevoie** [comment]
Un usager nous fait un retour sur ce manquement: "Il serait utile **d'intégrer les aides aux contrats PEC**. Pour estimer facilement le coût d'un salarié pour une petite association."
